### DISCUSSION TITLE
Allow Death Knight creation on the PTR.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#5948] [Suggestions]
Pretty self-explanatory. Having later content being available to at least take a peek at on the PTR for things like tuning or bug testing can be quite helpful, such as the various tests on the PTR of upcoming raid tiers (Kara, Gruul, and Mag come to mind). Obviously there is good reasoning to have Death Knights locked out on the live server (tuning, performance, progression, etc.), but I can't see a good reason to lock them out on the PTR as well. Sure, their starting experience might cause crashes here and there, but better a year in advance on the PTR than multiple times on day one release on the live realm.

**Comment by Vyperhand** [comment]
Emphatically agree.  :)

**Comment by Never6** [comment]
Yes

**Comment by elthehablo** [comment]
Hard agree. Would be nice to play with dks in 70 raid setting too when testing the new raids :)

**Comment by Contrubius** [comment]
Do we have a confirmation DK starting zone quests are scripted in the first place? Most of their spells and talent points come from zone quests.

**Reply by heyitsbench** [reply]
I can personally confirm that at least one of the last times I tried, I was able to bring a death knight all the way through the starting experience and into the main world on AC. I can also confirm on my most recent venture over there that it was [crashing on a certain quest](https://github.com/azerothcore/azerothcore-wotlk/issues/14631), but that has been fixed since I reported it.

**Comment by Nyeriah** [comment]
Honestly, there's not much benefit from this other than allowing players to test them but we won't be able to focus on addressing DK issues so its a moot point anyway. I am only afraid it might lead to confusion later down the road. For raid testing we mostly definitely won't want death knights as we will also be testing the tuning.

I believe this is something to be considered when we are closer to releasing Wrath, like during the BT bracket or so.

**Reply by heyitsbench** [reply]
May I ask why Northrend content is available on the PTR if the devs can't focus on that? Why is Hyjal available on the PTR if that's not even close to being released? I don't see what confusion it would cause, seeing as it's pretty obvious when DK's are releasing on the live realm.

If tuning testing is something that needs to happen, DKs are pretty easy to spot to invalidate a test being done, not that that should happen anyway if the group testing knows what it is they're testing. It's a heck of a lot easier to spot than, for example, Sunwell gear during a Kara test.

**Reply by Nyeriah** [reply]
Northrend being released is a collateral effect as we needed to enable the Wrath expansion in order to enable outlands RDFs. That’s all
